roles/istio/tasks/certificates.yaml

```diff
@@ -0,0 +1,137 @@
+---
+
+################### Start Get Istio External IP ###################
```

Comment:
Do we need the comments for each section? These should be self explanatory. 
If you think comments are required here we could have a paragraph / bullet points at the beginning of the file maybe?